apps/dashboard/src/main.ts

```diff
@@ -84,11 +81,6 @@ app.component('Steps', Steps);
 app.component('Calendar', Calendar);
 app.component('ConfirmDialog', ConfirmDialog);
 
-populateStoresFromToken(apiService).then(() => {
-    app.mount('#app');
-}).catch(() => {
-    clearTokenInStorage();
-    const authStore = useAuthStore();
-    authStore.logout();
-    app.mount('#app');
-});
+await beforeLoad();
+
+app.mount('#app');
```

Comment:
Top level await is not allowed, use a `.then` instead.

(this is also why you have a build error)

@SuperVK